walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -48,24 +51,18 @@
       return e1when.start() - e2when.start();
     };
 
-    // provides sorting
-    ArrayList<Event> requiredEvents = new ArrayList<>();
-
     // check if request for a valid duration
     if (request.getDuration() > TimeRange.WHOLE_DAY.duration()) {
       return Arrays.asList();
     }
 
+    ArrayList<Event> requiredEvents = new ArrayList<>();
+    ArrayList<Event> optionalEvents = new ArrayList<>();
```

Comment:
Don't move `optionalEvents` here, it's moved further away from where it's first used.

Reply:
sure, we can just decline this PR, I did this cause personally I like stream api, but im fine with the either way

Reply:
Sounds good, but thanks for putting thoughts into exploring a different approach, which is good initiative and I really appreciate that!